lib/utils/age.ts

```diff
@@ -0,0 +1,12 @@
+import { differenceInYears, format, isValid, parseISO } from "date-fns";
+
+export const isUnderEighteen = (dob: string) => {
+  const isValidDate = parseISO(dob);
```

Comment:
use the parseDate utility here, some dates could be in error set to 1900-01-01 when its meant to be null.